test/fixtures.yml

```diff
@@ -198,6 +198,10 @@ checkout_v2:
   client_id: CLIENT_ID_FOR_OAUTH_TRANSACTIONS
   client_secret: CLIENT_SECRET_FOR_OAUTH_TRANSACTIONS
 
+checkout_v2_api:
+  secret_key: sk_sbox_xxxxxxxxxxxxxxxxx
+  public_key: pk_sbox_xxxxxxxxxxxxxxxxx
```

Comment:
I don't think we need this, the `checkout_v2` entry in the fixtures has the same fields.

Reply:
Yes, but the values are different from the checkout_v2 and the chekout_v2_api.

could works add something like this

checkout_v2:
  secret_key: SECRET_KEY_FOR_BASIC_TRANSACTIONS
  client_id: CLIENT_ID_FOR_OAUTH_TRANSACTIONS
  client_secret: CLIENT_SECRET_FOR_OAUTH_TRANSACTIONS
  secret_api_key: sk_sbox_xxxxxxxxxxxxxxxxx
  secret_public_key: pk_sbox_xxxxxxxxxxxxxxxxx

Reply:
Ah, if they are different then what you are doing is fine. Maybe we call it `checkout_v2_token` so it's a bit more descriptive.

The only other concern is, if our users need to have 2 sets of credentials, then we should handle it as one set of credentials and call the ones used for store/unstore something else.